sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -254,8 +254,9 @@ private static synchronized void init(
     Sentry.globalHubMode = globalHubMode;
 
     final IScopes hub = getCurrentScopes();
-    // TODO new Scopes()
-    mainScopes = new Hub(options);
+    final IScope rootScope = new Scope(options);
```

Comment:
This bit in `init` will have to be replaced in a follow up PR.

Reply:
This code region has changed multiple times. https://github.com/getsentry/sentry-java/pull/3362 has the most recent state (for now :D)